script.xbmc.boblight/addon.xml

```diff
@@ -0,0 +1,57 @@
+    <summary lang="pt_PT">Ligar saída do Kodi a boblight.</summary>
+    <description lang="ca_ES">Connecta la sortida del Kodi a un mòdul extern de boblight</description>
+    <description lang="de_DE">Verbinde Kodi Ausgabe mit boblight Backend</description>
+    <description lang="el_GR">Σύνδεση της εξόδου Kodi στο backend του boblight.</description>
+    <description lang="en_GB">Connect Kodi output to boblight backend.</description>
+    <description lang="gl_ES">Conectar a saída do Kodi ao motor de boblight.</description>
+    <description lang="he_IL">חבר פלט Kodi לשרת Boblight.</description>
+    <description lang="hr_HR">Poveži Kodi izlaz s boblight pozadinski.</description>
+    <description lang="hu_HU">Kodi kimenetének boblight háttérszolgáltatáshoz kapcsolása.</description>
+    <description lang="it_IT">Connetti l'output di Kodi al backend di boblight.</description>
+    <description lang="ko_KR">Kodi 출력을 boblight 백엔드에 연결</description>
+    <description lang="nl_NL">Verbind Kodi output naar boblight backend.</description>
+    <description lang="pl_PL">Podłącz wyjście Kodi do konsoli boblight.</description>
+    <description lang="pt_PT">Ligar saída do Kodi à backend boblight.</description>
+    <disclaimer lang="ca_ES">Necessites tindre el binari correcte perquè el complement funcioni</disclaimer>
+    <disclaimer lang="de_DE">Du musst das korrekte Binary Addon installiert haben, damit diese Funktion unterstützt wird.</disclaimer>
+    <disclaimer lang="el_GR">Χρειάζεται να έχετε εγκατεστημένο ένα κατάλληλο δυαδικό πρόσθετο για να δουλέψει.</disclaimer>
+    <disclaimer lang="en_GB">You need to have the correct binary addon installed for this to work.</disclaimer>
+    <disclaimer lang="gl_ES">Precisa ter instalado o complemento binario correcto para que funcione.</disclaimer>
+    <disclaimer lang="he_IL">ההרחבה הבינרית המתאימה צריכה להיות מותקנת ע"מ שזה יפעל.</disclaimer>
+    <disclaimer lang="hr_HR">Morate imati ispravan binarni dodatak instaliran da bi ovo radilo.</disclaimer>
+    <disclaimer lang="hu_HU">Telepítve kell legyen a megfelelő bináris könyvtár ehhez a kiegészítőhöz.</disclaimer>
+    <disclaimer lang="it_IT">E' necessario avere l'addon binario giusto perchè questo funzioni.</disclaimer>
+    <disclaimer lang="ko_KR">동작하려면 바이너리 애드온이 설치되어 있어야 합니다.</disclaimer>
+    <disclaimer lang="nl_NL">Je moet de juiste binaire add-on hebben geïnstalleerd voor dit om te werken.</disclaimer>
+    <disclaimer lang="pl_PL">W celu poprawnego działania musisz mieć zainstalowaną odpowiednią bibliotekę dodatku.</disclaimer>
+    <disclaimer lang="pt_PT">É necessário instalar o addon binário correcto para que isto funcione.</disclaimer>
+    <platform>all</platform>
+    <website>http://code.google.com/p/boblight/</website>
+    <source>https://github.com/bobo1on1/script.xbmc.boblight</source>
+    <forum>http://forum.kodi.tv/showthread.php?tid=116331</forum>
+    <email></email>
+    <assets>
```

Comment:
license tag is missing, use GPL-2.0-only or GPL-2.0-or-later (spdx identifiers)

Reply:
I think GPL-3.0-or-later is correct here, e. g. as specified in `resources/lib/boblight.py`.

Reply:
It doesn't match with the provided licence.txt which is gpl2